Documentation/sources/content-card-ui/public-classes/ContentProvider/aepuicontentprovider.md

```diff
@@ -0,0 +1,40 @@
+# Interface - AepUIContentProvider
```

Comment:
Do the docs in this repo have this precedent of specifying the type of the API component (sealed class, interface, data class etc)? I prefer that we let the example demonstrate what that is.

Reply:
we do have it in the repo already but in a more general classification. for example, Enum - MessagingEdgeEventType or Message (Interface). i can remove adding the type when these docs are added to the aep-mobile docs.